src/vs/workbench/contrib/views/browser/treeView.ts

```diff
@@ -462,8 +463,16 @@ export class TreeView extends Disposable implements ITreeView {
 				return;
 			}
 			const selection = this.tree!.getSelection();
-			if ((selection.length === 1) && selection[0].command) {
-				this.commandService.executeCommand(selection[0].command.id, ...(selection[0].command.arguments || []));
+			const command = selection[0].command;
```

Comment:
I think that `getSelection` can return an empty array.

Reply:
Thanks, fixed.